### ISSUE TITLE
FNX-5175 ⁃ IllegalStateException: bookmark_name_edit must not be null

### ISSUE THREAD

**Issue by kbrosnan** [mozilla-mobile/fenix#1675]
https://sentry.prod.mozaws.net/operations/fenix/issues/5554257/

```
IllegalStateException: bookmark_name_edit must not be null
    at org.mozilla.fenix.library.bookmarks.edit.EditBookmarkFragment$onPause$1.invokeSuspend(EditBookmarkFragment.kt:5)
...
(5 additional frame(s) were not displayed)

bookmark_name_edit must not be null
```

This sounds like a simple null check is needed.

**Comment by colintheshots** [comment]
The issue is a matter of timing, a race condition. This is getting fixed incidentally by PR#1708. I noticed that we sometimes weren't getting bookmark moves saved and decided to move the logic from onPause to onResume to ensure we perform the work early enough.

**Comment by sv-ohorvath** [comment]
The crash was fixed. Verified on the 4/21 build.
